pkg/providers/instancetype/types.go

```diff
@@ -436,28 +436,28 @@ func habanaGaudis(info *ec2.InstanceTypeInfo) *resource.Quantity {
 	return resources.Quantity(fmt.Sprint(count))
 }
 
-func efas(info *ec2.InstanceTypeInfo) *resource.Quantity {
-	count := int64(0)
+func efas(info ec2types.InstanceTypeInfo) *resource.Quantity {
+	count := int32(0)
 	if info.NetworkInfo != nil && info.NetworkInfo.EfaInfo != nil {
-		count = lo.FromPtr(info.NetworkInfo.EfaInfo.MaximumEfaInterfaces)
+		count = *info.NetworkInfo.EfaInfo.MaximumEfaInterfaces
```

Comment:
Can maximum interfaces be `nil` here?

Reply:
We need to go through the describe instance type value and valudate